BaseApi.js

```diff
@@ -144,6 +145,9 @@ const API = {
     const url = module_url.replace('$ideaPk', ideaPk);
     return makePutRequest(url, data, token);
   },
+  postReport(content_type, object_pk, description, token = null) {
+    return makePostRequest(endpoints.report, {content_type, object_pk, description}, token);
```

Comment:
Maybe you could pass data as a dict here and already handle that in handleSubmit? So I mean here just 
```postReport(data, token=null)``` and makePostRequest(endpoint, data, token) and in handleSubmit API.postReport({content_type, object_pk, description}, token).

Reply:
thanks for explaining! I didn't get the difference in the requests before